site/content/docs/5.2/components/button-group.md

```diff
@@ -40,6 +40,12 @@ These classes can also be added to groups of links, as an alternative to the [`.
 
 ## Mixed styles
 
+{{< ods-incompatibility-alert >}}
+This variant **using and mixing colored button backgrounds** should not be used because it does not respect the Orange Design System specifications. In button groups, you should only use the button variant that using `.btn .btn-secondary`.
```

Comment:
```suggestion
This variant **using and mixing colored button backgrounds** should not be used because it does not respect the Orange Design System specifications. In button groups, you should only use the button variant that use `.btn .btn-secondary`.
```

Reply:
```suggestion
This variant **using and mixing colored button backgrounds** should not be used because it does not respect the Orange Design System specifications. In button groups, you should only use the button variant that uses `.btn .btn-secondary`.
```